### DISCUSSION TITLE
Infineat watch face is missing heart rate monitor icon/reading.

### DISCUSSION THREAD

**Discussion by apoorv569** [InfiniTimeOrg/InfiniTime#1469] [Ideas]
### Verification

- [X] I searched for similar bug reports and found none was relevant.

### What happened?

The heart rate icon/reading is missing.

### What should happen instead?

Heart rate icon/reading should appear as in other watch faces.

### Reproduction steps

Change the watch face to Infineat.

### Version

v1.11

### Companion app

Gadgetbridge

**Comment by JF002** [comment]
This could be the result of an artistic choice by the author of the watchface. @dmlls, what do you think?

**Comment by dmlls** [comment]
@apoorv569 @JF002 Hi, it was indeed a design choice, briefly discussed here: https://github.com/dmlls/InfiniTime/pull/3#issuecomment-1072860833.

In any case, if users think this is a must, I'm open to reconsidering :)

**Comment by apoorv569** [comment]
> @apoorv569 @JF002 Hi, it was indeed a design choice, briefly discussed here: [dmlls#3 (comment)](https://github.com/dmlls/InfiniTime/pull/3#issuecomment-1072860833).
> 
> In any case, if users think this is a must, I'm open to reconsidering :)

I see. Well we can definitely wait for some feedback from other users.

**Comment by InFerYes** [comment]
I find the HR monitor to be more useful if it can be read from the watch face.

**Comment by JF002** [comment]
Thanks for your feedback @dmlls ! 

@apoorv569 If you don't mind, I'll move this post to a [discussion](https://github.com/InfiniTimeOrg/InfiniTime/discussions) to let everyone interested can give their opinion.
